kt/godot-library/src/main/kotlin/godot/core/IReadOnlyVariantArray.kt

```diff
@@ -0,0 +1,28 @@
+package godot.core
+
+interface IReadOnlyVariantArray<T> : Collection<T> {
```

Comment:
I wouldn't use a "I" prefix here.  Because it means that users will have to manipulate the type IReadOnlyVariantArray or VariantArray side by side. This is not consistent.